### ISSUE TITLE
Runbook for COVID 19 vaccines to lighthouse

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#3824]
This is a sprint goal, but some items depend on other teams. 

====

This is the runbook of what must happen in order to get Covid Vaccine data to lighthouse.

- [x] Get final post format from Lighthouse ([thread here](https://dsva.slack.com/archives/C0FQSS30V/p1607551861482000))
- [x]  Update our CMS post to match  format
- [x]  Test and validate with lighhouse
- [x] Get final facility service list from Dave Conlon for Pfizer
- [x] Get final facility service list from Dave Conlon for Moderna (nice to have, not necessary for proof of concept)
   - [x] Add list to CMS  https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4002
- [x] Update documentation
- [x] Merge [post code PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3794) and release to production.
- [x] Set Covid 19 Vaccines Taxonomy term to published
     This has been done by Dave  
      ![image](https://user-images.githubusercontent.com/5752113/102663299-8c2dd300-414e-11eb-900a-16e874ba6b12.png)

## When Dave gives the greenlight:
- [x]  run system, system service, and facility service migrations on production.
 by going here `/admin/structure/migrate/manage/vamc/migrations/facility_health_care_service_covid/execute` and click the execute button
- [x] go to `/admin/content/bulk?title=covid-19&type=health_care_local_health_service&moderation_state=All&taxonomy_entity_index_tid_depth=All`  and bulk publish the nodes
- [x] run the queue /admin/config/post-api/queue
- [x]  Validate  with lighthouse that data is showing on api.va.gov.

**Comment by swirtSJW** [comment]
I updated the documentation.  Moving this back to TODO as we wait on the API.

**Comment by swirtSJW** [comment]
We have some new directives from Dave here https://dsva.slack.com/archives/CPQFCQKEH/p1608759676269200   I will spin up associated tickets in the morning.

**Comment by kevwalsh** [comment]
This was not completed due to the external dependencies. Moving to Sprint 23, still a sprint goal, still the same external dependencies.

**Comment by swirtSJW** [comment]
Endpoint was provided today.  It was tested and worked for both adding(publishing) and removing(archiving).

**Comment by swirtSJW** [comment]
Dave confirmed the data we have for Phizer is still the most up to date
![image](https://user-images.githubusercontent.com/5752113/104069067-6175ee00-51d3-11eb-95de-d4a8f2427fb3.png)

**Comment by swirtSJW** [comment]
Update.   We are not running the import and publishing the covid vaccine services until futher notice from Dave.
![image](https://user-images.githubusercontent.com/5752113/104232148-ea806580-541d-11eb-875a-dc26433852db.png)

**Comment by swirtSJW** [comment]
This one was excluded from the data `'691C';'VA Greater Los Angeles health care';` because we could not determine what facility it belonged to.  There are multiple 691 in LA but none with the is 691C.   This will have to be added by hand when we get clarification.

**Comment by swirtSJW** [comment]
System service migration run.  
![image](https://user-images.githubusercontent.com/5752113/104511965-7c71a500-55bb-11eb-902c-0b6bb779db42.png)

Facility service migration run 
![image](https://user-images.githubusercontent.com/5752113/104512996-00785c80-55bd-11eb-9a95-f293e82c30b8.png)

Published
![image](https://user-images.githubusercontent.com/5752113/104513495-bf347c80-55bd-11eb-8bc6-ca7cc126a82d.png)

and Queue processed.

**Comment by swirtSJW** [comment]
Confimred we are good with lighthouse.  They have all the data they are supposed to
![image](https://user-images.githubusercontent.com/5752113/104515410-c610be80-55c0-11eb-8f35-cabd2a40930c.png)


Surfaced that pittsburgh data ended up in the migration and was not supposed to.  It is removed with this PR
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4005

**Comment by swirtSJW** [comment]
All boxes are ticked.  Calling this done.
